src/compas_rhino/geometry/surfaces/surface.py

```diff
@@ -189,7 +189,7 @@ def from_plane(cls, plane, box):
         return cls.from_rhino(rhino_surface)
 
     @classmethod
-    def from_frame(cls, frame, u_interval, v_interval, uv_degrees, uv_point_counts):
+    def from_frame(cls, frame, u_interval, v_interval, uv_degrees=(1, 1), uv_point_counts=(2, 2)):
```

Comment:
would it not make more sense to do `u_degree=1, v_degree=1, u_count=2, v_count=2`? since this is how it is done for the intervals as well...

Reply:
agree. done.